webapp/src/filtering/filtering.tsx

```diff
@@ -1,7 +1,17 @@
+import { responsePathAsArray } from "graphql";
 import { useEffect, useRef, useState } from "react"
-import replayArray from "./replays"
+import rankImages from './ranks';
+
+const replayArray = await fetch("http://localhost:5001/parse/replayList").then((response) => response.json())
+const arenaArray = await fetch("http://localhost:5001/parse/all/arena").then((response) => response.json())
+const rankArray = await fetch("http://localhost:5001/parse/all/ranking").then((response) => response.json())
+const durationArray = await fetch("http://localhost:5001/parse/all/duration").then((response) => response.json())
+const seasonArray = await fetch("http://localhost:5001/parse/all/season").then((response) => response.json())
+const gamemodeArray = await fetch("http://localhost:5001/parse/all/gamemode").then((response) => response.json())
+const gametypeArray = await fetch("http://localhost:5001/parse/all/gametype").then((response) => response.json())
 
 export default function SearchReplays({width = 1280, height = 720}: {width?: number, height?: number}) {
```

Comment:
For future reference, it'll probably be better to separate so that one file is one component